server/controllers/inventory/inventory/groups.js

```diff
@@ -59,7 +59,10 @@ function update(record, identifier) {
  */
 function getGroups(uid) {
   const sql = `
-    SELECT BUID(uuid) AS uuid, code, name, sales_account, cogs_account, stock_account, expires, unique_item
+    SELECT 
+      BUID(uuid) AS uuid, code, name, sales_account,
+      cogs_account, stock_account, expires, unique_item,
+      should_track_consumption, should_track_expiration
```

Comment:
Here is a suggestion for the variable names:
   should_track_consumption ==> tracking_consumption
   should_track_expiration ==> tracking_expiration
The "should_" versions seem a bit long.  And the "tracking_" ones have the same boolean sense so only the name would be changed.